lib/tbot/workloadidentity/workloadattest/container/container.go

```diff
@@ -0,0 +1,101 @@
+/*
+ * Teleport
+ * Copyright (C) 2025  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package container
+
+import (
+	"path"
+	"strconv"
+
+	"github.com/gravitational/trace"
+	"k8s.io/utils/mount"
+)
+
+// Rootfulness describes whether a container was started by an unprivileged user
+// as in "rootless" Podman or by root. It's a best guess based on information
+// gleaned from procfs, not authoratative.
+type Rootfulness int
+
+const (
+	// RootfulnessUnknown means we were unable to infer whether the container is
+	// rootless or not.
+	RootfulnessUnknown Rootfulness = iota
+
+	// Rootful means the container was probably started by root.
+	Rootful
+
+	// Rootless means the container was probably started by an unprivileged user.
+	Rootless
+)
```

Comment:
My original thought behind inferring the container's "rootfulness" was that we could use it to determine the correct socket location (i.e. `/run` or `$XDG_RUNTIME_DIR`) but as mentioned in the PR description I decided it's better to leave this configuration up to the user.

We could remove this code now, but it might be helpful for debugging or when it comes to implementing the Docker attestor. WDYT?